pom.xml

```diff
@@ -143,9 +143,51 @@
                             <goals>
                                 <goal>jandex</goal>
                             </goals>
+                            <!-- phase is 'process-classes by default' -->
+                            <configuration>
+                                <fileSets>
+                                    <fileSet>
+                                        <directory>${project.build.directory}</directory>
+                                    </fileSet>
+                                </fileSets>
+                            </configuration>
                         </execution>
                     </executions>
                 </plugin>
+
+                <!--This plugin's configuration is used to store Eclipse m2e settings
+                    only. It has no influence on the Maven build itself. -->
+                <plugin>
+                    <groupId>org.eclipse.m2e</groupId>
+                    <artifactId>lifecycle-mapping</artifactId>
+                    <version>1.0.0</version>
+                    <configuration>
+                        <lifecycleMappingMetadata>
+                            <pluginExecutions>
+                                <pluginExecution>
+                                    <pluginExecutionFilter>
+                                        <groupId>
+                                            org.codehaus.mojo
+                                        </groupId>
+                                        <artifactId>
+                                            native2ascii-maven-plugin
+                                        </artifactId>
+                                        <versionRange>
+                                            [1.0-beta-1,)
```

Comment:
native2ascii-maven-plugin hasznalt verzioja 2.1.0 igy ide is akkor az kellene

Reply:
köszi szépen!